app/views/admin/films/_film.html.erb

```diff
@@ -0,0 +1,51 @@
+<tr>
+  <td><%= film.image %></td>
+  <td><%= link_to film.name, admin_film_path(film) %></td>
+  <td><%= film.play_time %></td>
+  <td><%= film.link_trailer %></td>
+  <td><%= film.avg_rate %></td>
+  <td><%= film.describe %></td>
+  <td>
+    <button type="button" class="btn btn-info" data-toggle="modal" data-target="#myModal-<%= film.id %>"><%= t ".edit_movie" %></button>
+    <div class="modal fade" id="myModal-<%= film.id %>" role="dialog">
```

Comment:
Mỗi một film được in ra sẽ cần 1 modal. Vậy 100 film sẽ cần 100 modal. Như vậy sẽ không hiệu quả. Các bạn chỉ nên có 1 modal, khi dùng tới sẽ render ra content cho phù hợp